src/components/Logo.js

```diff
@@ -0,0 +1,22 @@
+import {Image, StyleSheet, Text, View} from 'react-native';
```

Comment:
As logo is a common component, so I would appreciate if we can create a common folder in our project and move all components there.

Reply:
mark it as resolved